nixos/modules/services/x11/desktop-managers/gnome3.xml

```diff
@@ -0,0 +1,250 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+         xmlns:xlink="http://www.w3.org/1999/xlink"
+         xml:id="chap-gnome">
+ <title>GNOME Desktop</title>
+ <!-- TODO: short description of what gnome is and links to external docs -->
+ <para>
+ </para>
+
+ <section xml:id="sec-gnome-enable">
+  <title>Enabling GNOME</title>
+
+  <para>
+   All of the core apps, optional apps, and games from GNOME are available.
+  </para>
+
+  <para>
+   To enable the GNOME desktop use:
+  </para>
+
+<programlisting>
+<xref linkend="opt-services.xserver.desktopManager.gnome3.enable"/> = true;
+</programlisting>
+
+  <para>
+   and it is also recommended to enable:
+  </para>
+
+<programlisting>
+<xref linkend="opt-services.xserver.displayManager.gdm.enable"/> = true;
+</programlisting>
+
+  <para>
+   as the display-manager.
+  </para>
+
+  <para>
+   The default applications used in NixOS are very minimal, inspired from the defaults used in <link xlink:href="https://gitlab.gnome.org/GNOME/gnome-build-meta/-/blob/3.34.4/elements/core/meta-gnome-core-utilities.bst">gnome-build-meta</link>.
+  </para>
+
+  <section xml:id="sec-gnome-without-the-apps">
+   <title>GNOME without the apps</title>
+
+   <para>
+    If you’d like to only use the GNOME desktop and not the apps, you can disable them with:
+   </para>
+
+<programlisting>
+<xref linkend="opt-services.gnome3.core-utilities.enable"/> = false;
+</programlisting>
+
+   <para>
+    and none will be installed.
+   </para>
+
+   <para>
+    If you’d only like certain omissions you can use <xref linkend="opt-environment.gnome3.excludePackages"/>.
+    Note, this only removes things installed in the gnome3 module from <xref linkend="opt-environment.systemPackages"/>.
+    Certain applications require an entire nixos module, like <literal>evince</literal> or <literal>geary</literal>.
+    This means using <xref linkend="opt-environment.gnome3.excludePackages"/> won't disable them properly.
+    Those default applications are listed <link xlink:href="https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/x11/desktop-managers/gnome3.nix#L364">here</link>,
+    and you should set the value of the enable options to <literal>false</literal> to disable them.
```

Comment:
The option has been extended to core utilities in https://github.com/NixOS/nixpkgs/pull/98510. Though services or GNOME core still cannot be disabled this way.